src/ifcblenderexport/blenderbim/export_ifc.py

```diff
@@ -824,6 +876,10 @@ def export(self):
         self.relate_objects_to_types()
         self.relate_objects_to_qtos()
         self.relate_objects_to_psets()
+
+        self.relate_voids_elements()
+        self.relate_fills_elements()
```

Comment:
Please rename function to `self.relate_opening_elements_to_fillings()` - like above, this is to copy the IFC jargon as closely as possible.